latest master fails to build on ubuntu 18.04.

Affects versions :

    OS: ubuntu 18.04

Describe the bug
after running:

`cmake`

end up with:

```
xournalpp/build/external/Source/ghcFilesystem_git/include/ghc/filesystem.hpp: In instantiation of ‘ghc::filesystem::path& ghc::filesystem::path::append(const Source&) [with Source = std::basic_string_view<char>]’:
/home/pgroszko/xournalpp/build/external/Source/ghcFilesystem_git/include/ghc/filesystem.hpp:2406:25:   required from ‘ghc::filesystem::path& ghc::filesystem::path::operator/=(const Source&) [with Source = std::basic_string_view<char>]’
/home/pgroszko/xournalpp/src/util/PathUtil.cpp:163:18:   required from here
/home/pgroszko/xournalpp/build/external/Source/ghcFilesystem_git/include/ghc/filesystem.hpp:2412:48: error: no matching function for call to ‘toUtf8(const std::basic_string_view<char>&)’
     return this->operator/=(path(detail::toUtf8(source)));
                                  ~~~~~~~~~~~~~~^~~~~~~~
/home/pgroszko/xournalpp/build/external/Source/ghcFilesystem_git/include/ghc/filesystem.hpp:1573:20: note: candidate: template<class charT, class traits, class Alloc, typename std::enable_if<(sizeof (charT) == 1), int>::type size> std::__cxx11::string ghc::filesystem::detail::toUtf8(const std::__cxx11::basic_string<_CharT, _Traits, _Alloc>&)
 inline std::string toUtf8(const std::basic_string<charT, traits, Alloc>& unicodeString)
                    ^~~~~~
/home/pgroszko/xournalpp/build/external/Source/ghcFilesystem_git/include/ghc/filesystem.hpp:1573:20: note:   template argument deduction/substitution failed:
/home/pgroszko/xournalpp/build/external/Source/ghcFilesystem_git/include/ghc/filesystem.hpp:2412:48: note:   ‘const std::basic_string_view<char>’ is not derived from ‘const std::__cxx11::basic_string<_CharT, _Traits, _Alloc>’
     return this->operator/=(path(detail::toUtf8(source)));
[...]
src/util/CMakeFiles/util.dir/build.make:182: recipe for target 'src/util/CMakeFiles/util.dir/PathUtil.cpp.o' failed
make[2]: *** [src/util/CMakeFiles/util.dir/PathUtil.cpp.o] Error 1
CMakeFiles/Makefile2:465: recipe for target 'src/util/CMakeFiles/util.dir/all' failed
make[1]: *** [src/util/CMakeFiles/util.dir/all] Error 2
Makefile:151: recipe for target 'all' failed
make: *** [all] Error 2
```

The build does not complete. 
This has been a problem for some number of weeks now... maybe two or three. 
Thanks.

[CMakeError.log](https://github.com/xournalpp/xournalpp/files/5697513/CMakeError.log)
[CMakeOutput.log](https://github.com/xournalpp/xournalpp/files/5697514/CMakeOutput.log)

--- a/cmake/find/FindFilesystem.cmake
+++ b/cmake/find/FindFilesystem.cmake
@@ -276,6 +276,7 @@ elseif (NOT _found AND "ghc" IN_LIST want_components)
     ExternalProject_Get_Property(${GHC_FILESYSTEM} SOURCE_DIR)
     file(MAKE_DIRECTORY "${SOURCE_DIR}/include")
     target_include_directories(std_filesystem INTERFACE "${SOURCE_DIR}/include")
+    target_compile_definitions(std_filesystem INTERFACE -DGHC_FILESYSTEM)
 
     add_library(std::filesystem ALIAS std_filesystem)
 endif ()

--- a/src/util/PathUtil.cpp
+++ b/src/util/PathUtil.cpp
@@ -10,7 +10,7 @@
 #include "XojMsgBox.h"
 #include "i18n.h"
 
-#ifdef __APPLE__
+#ifdef GHC_FILESYSTEM
 // Fix of ghc::filesystem bug (path::operator/=() won't support string_views)
 constexpr auto const* CONFIG_FOLDER_NAME = "xournalpp";
 #else

